torch/nn/modules/activation.py

```diff
@@ -279,6 +279,36 @@ def __repr__(self):
         return self.__class__.__name__ + inplace_str
 
 
+class GLU(Module):
+    """Applies the gated linear unit function :math:`{GLU}(a, b)= a \otimes \sigma(b)`
+    where `a` is the first half of the input vector and `b` is the second half.
```

Comment:
I don't see a reason why do we force the input to be a single vector. The interface would be much more logical if we were to accept `a` and `b` separately, as in the equation that's shown here.